tests/model/units_import.cpp

```diff
@@ -146,6 +147,23 @@ TEST(UnitsImport, nonExistentUrl)
     EXPECT_EQ(e, a);
 }
 
+TEST(UnitsImport, noNameAttribute)
+{
+    const std::string in =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/2.0#\" name=\"model\">\n"
+        "  <import xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"http://someplace.world/cellml/model.xml\">\n"
+        "    <units units_ref=\"per_mole\"/>\n"
+        "  </import>\n"
+        "</model>\n";
+
+    libcellml::ParserPtr p = libcellml::Parser::create();
+
+    libcellml::ModelPtr m = p->parseModel(in);
+
+    EXPECT_EQ(size_t(1), p->errorCount());
```

Comment:
Check the exact issue.

Reply:
[8d263f5](https://github.com/cellml/libcellml/pull/1225/commits/8d263f5fc4e1177e26a822d743ac1b9c49c834a7)